### PR TITLE
Add Code Quality and security doc

### PR DESCRIPTION
**Pull request by jmertic**
Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

### CODE REVIEW

File: CodeQualitySecurity.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- [SonarCloud](https://sonarcloud.io/dashboard?id=odpi_egeria) to continously track code quality, including bugs, code smells, and potential security issues.
+- <a href="https://www.yourkit.com/java/profiler/">YourKit Java Profiler</a> for doing code profiling of Java components. YourKit supports open source projects with innovative and intelligent tools for monitoring and profiling Java and .NET.
+- [Community Bridge Vulnerability Detection](https://security.communitybridge.org) for proactive assessment of vulnerabilities in dependent libraries and packages.
+- [GitHub Security Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories) for privately discussing and managing security issues that come through (egeria-security@odpi.org) and other confidential channels.
```
Comment (cmgrote) [inline CodeQualitySecurity.md]: Would suggest changing this to:

```
- [GitHub Security Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories) for internally triaging security issues that come through (egeria-security@odpi.org) and other confidential channels before publishing them for broader community awareness.
```

(The intent of the security advisories is really as much about making the broader community aware as privately handling the discussions.)

**Comment by jmertic** [comment]
Thank you @cmgrote  - incorporated your feedback now.

**Comment by jmertic** [comment]
@planetf1 Thanks for point that out - would you recommend we roll this into those docs, those docs into this, or something else?

**Comment by planetf1** [comment]
I actually like the filename you chose, and the conciseness of the information which is especially suitable for anyone coming along to find out high level info about the security of egeria.

In most cases the other docs get into more detail - but even then there are overlaps between those two.

One other thing that's missing is pointing to this file from the overall README in developer resources

If it makes it easier I'd be happy we merge this, and then I can try and consolidate the three files with a view to keeping the principle of
 - the high level security overview/reports
 - more detailed process info for developers

Note though that currently you're missing the required header to pass our license check. Can you add
```
<!-- SPDX-License-Identifier: CC-BY-4.0 -->
<!-- Copyright Contributors to the ODPi Egeria project. -->
```
to the top of the file?

**Comment by jmertic** [comment]
Thanks @planetf1 - added the headers now :-)

I'll leave it to you whether to merge now or fix the other docs at the same time...

**Comment by planetf1** [comment]
> Thanks @planetf1 - added the headers now :-)
> 
> I'll leave it to you whether to merge now or fix the other docs at the same time...

Will merge and open up new issue

**Pull request merged**